src/ApplicationInsights.Kubernetes/Debuggings/Logger.cs

```diff
@@ -0,0 +1,107 @@
+using System.Diagnostics;
+using System.Diagnostics.Tracing;
+using System.Globalization;
+
+namespace Microsoft.ApplicationInsights.Kubernetes.Debugging
+{
+    /// <summary>
+    /// Diagnostic Source of Application Insights for Kubernetes.
+    /// </summary>
+    public sealed class Logger
+    {
+        private readonly DiagnosticSource _innerSource;
+
+        /// <summary>
+        /// Diagnostic source listener subscribe point.
+        /// </summary>
+        /// <returns></returns>
+        public DiagnosticListener Observer => (DiagnosticListener)this._innerSource;
```

Comment:
This is actually a bad name that makes things confusing. `DiagnosticListener` is __not__ an `Observer`. Instead, it enables `Observers` to subscribe to event streams. 

Suggest renaming this property to `Listener`.

Reply:
Totally!